### ISSUE TITLE
Clean preprocessing with held out test set

### ISSUE THREAD

**Issue by jjm82** [hunterchewitt-usc/LLM---Detect-AI-Generated-Text#12]
- Choose best data set from Kaggle
- create new file preprocessing.py
- put dataset into this file and hold out a randomly sampled test set which is balanced
- i.e. output a train/validate set and a held-out test set
- upload these files to the folder for permanent use (training/validate for CV and hyper-parameter tuning, test for final evaluation)
